locals.tf

```diff
@@ -18,5 +18,9 @@ locals {
   }), var.additional_script_path == null ? "" : file("${var.additional_script_path}")))
 
   additional_template_file = var.additional_script_uri != null ? format("%s%s%s", "[ ", "\"${var.additional_script_uri}\"", " ]") : "\"\""
+
+  #  Run command settings
+
```

Comment:
Should there be some locals here? Looks like something is missing

Reply:
missed to remove that comment, had used locals to test something but then removed them :)